Production_scripts/Create_costs/costs-care_home.R

```diff
@@ -29,40 +30,48 @@ fs::file_copy(get_ch_costs_path(),
   overwrite = TRUE
 )
 
-## Read excel data
-ch_costs_data <- readxl::read_xlsx(
-  paste0(get_slf_dir(), "/Costs/CH_Costs.xlsx")
-)
+## Read costs from the CHC Open data
+ch_costs_data <- phsopendata::get_resource(
+  res_id = "4ee7dc84-ca65-455c-9e76-b614091f389f",
+  col_select = c("Date", "KeyStatistic", "CA", "Value")
+) %>%
```

Comment:
this did not work for me, error with `col_select` is there a missing package? 

alternative that works is:
```suggestion
ch_costs_data <- phsopendata::get_resource(
  res_id = "4ee7dc84-ca65-455c-9e76-b614091f389f"
) %>%
select(Date, KeyStatistic, CA, Value) %>%
```

Reply:
`col_select` is  in the latest version of `phsopendata` if you get the new version it should work 😄